test/tracks.js

```diff
@@ -0,0 +1,372 @@
+
+'use strict';
+
+const assert = require('chai').assert;
+let generator = require('../src/backend/generator.js');
+let app = require('../src/app');
+let webdriver = require('selenium-webdriver');
+let trackPage = require('../src/selenium/trackPage.js');
+
+describe('app', () => {
+  describe('run', () => {
+    it('should run app', () => {
+      const expressApp = app.getApp();
+      assert.equal(expressApp.get('port'), (process.env.PORT || 5000));
+    });
+  });
+
+});
+
+describe('generate', function() {
+  describe('.create different event sites and copy assets of overview site', function() {
+    this.timeout(800000);
+
+    it('should generate the FOSSASIA Summit 2017', function(done) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Block must be padded by blank lines. (padded-blocks)](https://app.codacy.com/manual/triplez/open-event-webapp/pullRequest?prid=3226522)